integrations/docker/images/chip-build-crosscompile/Dockerfile

```diff
@@ -1,12 +1,17 @@
-ARG VERSION=latest
-FROM connectedhomeip/chip-build:${VERSION}
+FROM alpine:3.15
 
-COPY ubuntu-21.04-aarch64-sysroot.tar.xz /opt/ubuntu-21.04-aarch64-sysroot.tar.xz
+RUN apk --no-cache add \
+    git=2.34.1-r0 \
+    bash=5.1.8-r0 \
+    wget=1.21.2-r2
 
+WORKDIR /opt
 # Unpack the sysroot, while also removing some rather large items in it that
 # are generally not required for compilation
 RUN set -x \
-    && cd /opt \
+    && git clone --depth 1 https://chromium.googlesource.com/chromium/tools/depot_tools.git /opt/depot_tools \
+    && echo 'experimental/matter/sysroot/ubuntu-21.04-aarch64 latest' > ensure_file.txt \
```

Comment:
The location of this file was supposed to be a temporary solution, unsure if depending on something in 'experimental' is the best.

Would love opinions of others here.

Reply:
This does seem reasonable to me honest. Would save a lot of time :)

Reply:
What about including a `TODO` comment for the creation of an issue?

Reply:
TODO seems fair - maybe figuring out something that does not say 'experimental' in the name (I had no rights in other folders but this can be followed up when time allows)

Reply:
we can also delete depot_tools after the 'ensure'  if we really want to save more space.